src/Raven.Client/Documents/Operations/Backups/EncryptionSettings.cs

```diff
@@ -0,0 +1,7 @@
+namespace Raven.Client.Documents.Operations.Backups
+{
+    public class EncryptionSettings
+    {
+        public string Key { get; set; }
```

Comment:
That's the public key, right? Can we rename to be explicit about that in our API?

Reply:
This is the _secret_ key, not public key

Reply:
Okay. The issue description somehow made me think it's a public key - "1. The user supplies a public key and keeps the private key safe ..." http://issues.hibernatingrhinos.com/issue/RavenDB-10683

Anyway, I think it's better to be explicit so maybe call is `SecretKey` ?

Reply:
Yes, we changed the behavior along the way